code/modules/power/lighting/lighting.dm

```diff
@@ -302,8 +324,52 @@ var/global/list/obj/machinery/light/alllights = list()
 // attack with item - insert light (if right type), otherwise try to break the light
 
 /obj/machinery/light/attackby(obj/item/W, mob/living/user)
-	user.delayNextAttack(8)
+	if(ismultitool(W)) //RGB gamer lights
+		if(!rgb_upgrade)
+			to_chat(user, "There's nothing to connect \the [W] to on \the [src].")
+			return
+		var/which = alert(user, "What would you like to change?", "RGB Gamer lights", "Color", "Power", "Range")
```

Comment:
Please do not call them "gamer lights" in an IC text box.

Reply:
soulless

Reply:
ok